entries/c/carbonite.com.json

```
@@ -0,0 +1,14 @@
+{
+    "Canva": {
```

```suggestion
    "Carbonite": {
```
Thou shalt not copy-and-paste! :stuck_out_tongue_winking_eye:

---

:)

---

ups